pkgs/games/tiltfive/control-panel.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, lib
+, autoPatchelfHook
+, fetchurl
+, nixosTests
+
+, gtk3
+, glfw
+}:
+
+let srcs = import ./srcs.nix { inherit fetchurl; }; in
+stdenv.mkDerivation rec {
+  pname = "tiltfive-control-panel";
+  version = srcs.version;
+  src = srcs.driver;
+
+  buildInputs = [
+    gtk3 glfw
+  ];
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  installPhase = ''
+    cd tiltfive-control-panel_${version}_amd64/files/
+
+    dest=$out/opt/tiltfive/control-panel
+    pushd opt/tiltfive/control-panel
+
+    mkdir -p $dest/data
+    cp -r data/* $dest/data/
+
+    mkdir -p $dest/lib
+    install -Dm755 lib/* $dest/lib/
+
+    install -Dm755 control_panel $dest/
+
+
+    popd
+
+    mkdir -p $out/share/applications
+    install -Dm644 usr/share/applications/${pname}.desktop -t $out/share/applications
```

Comment:
Do not parameterize pname

Reply:
```
nixpkgs $ grep -r '{pname}.desktop' | wc -l
90
```

This seems to be a fairly common pattern in nixpkgs. Is there a concrete style recommendation against this?

Reply:
Right, but what is the reason to not do this? I'll gladly change things around, but I'd prefer to know what is the reasoning behind this so that I can then explain the same thing to others :).

Reply:
> This seems to be a fairly common pattern in nixpkgs. Is there a concrete style recommendation against this?

90 occurrences in a codebase of 80k+ packages? This is not "fairly common".

> Right, but what is the reason to not do this? 

1. it creates a useless dependency link, making static debugging a little more difficult.
2. sometimes `pname` is not perfectly parameterizable. E.g. the GitHub repository of `openmsx` is `openMSX/openMSX`. What we should do? Pass it under a obfuscated combination of string splitting?
3. being effectively a constant, using `pname` makes no real difference to understand the code. It is just harder to read and conveys nothing useful.